app/assets/v2/js/pages/profile-tribes.js

```diff
@@ -0,0 +1,272 @@
+/* eslint-disable no-loop-func */
+
+
+const loadDynamicScript = (callback, url, id) => {
+  const existingScript = document.getElementById(id);
+
+  if (!existingScript) {
+    const script = document.createElement('script');
+
+    script.src = url;
+    script.id = id;
+    document.body.appendChild(script);
+
+    script.onload = () => {
+      if (callback)
+        callback();
+    };
+  }
+
+  if (existingScript && callback)
+    callback();
+};
+
+(function($) {
+  // doc ready
+  $(() => {
+    Vue.use(VueQuillEditor);
```

Comment:
Is this loaded?

Reply:
i believe is loaded by the profile-vue.html